js/scripts.js

```diff
@@ -0,0 +1,12 @@
+var famaleNames = ['Magda', 'Ania', 'Gosia', 'Dominika'];
+var maleNames = ['Tomasz', 'Maciej', 'Bernard', 'Marcin'];
+var allNames = famaleNames.concat(maleNames);
+var newName = 'Marian';
+
+if (allNames.indexOf(newName) === -1 ) {
+	var x = allNames.push(newName);
```

Comment:
why you declare new variable? you don't use it in your later code, so this is unnecessary :)